app/views/job_invocations/_form.html.erb

```diff
@@ -45,31 +61,21 @@
             <%= radio_button_f targeting_fields, :targeting_type, :value => Targeting::STATIC_TYPE, :text => _(Targeting::TYPES[Targeting::STATIC_TYPE]) %>
             <%= radio_button_f targeting_fields, :targeting_type, :value => Targeting::DYNAMIC_TYPE, :text => _(Targeting::TYPES[Targeting::DYNAMIC_TYPE]) %>
           </div>
-	  <span class="help-inline"><%= popover(_('Explanation'),
-                                          _("Type has impact on when is the query evaulated to hosts.<br><ul><li><b>Static</b> - evaluates just after you submit this form</li><li><b>Dynamic</b> - evaluates just before the execution is started, so if it's planed in future, targeted hosts set may change before it</li></ul>")) %></span>
+
+          <span class="help-inline"><%= popover(_('Explanation'),
+                                        _("Type has impact on when is the query evaulated to hosts.<br><ul><li><b>Static</b> - evaluates just after you submit this form</li><li><b>Dynamic</b> - evaluates just before the execution is started, so if it's planed in future, targeted hosts set may change before it</li></ul>")) %>
+          </span>
         </div>
       <% end %>
 
       <% @composer.displayed_provider_types.each do |provider_type| %>
         <fieldset id="provider_<%= provider_type %>" class="provider_form">
-          <legend><%= _('Provider') + ': ' + provider_type %></legend>
+          <!--<legend><%= _('Provider') + ': ' + provider_type %></legend>-->
```

Comment:
History became legend. Legend became myth. And for two and a half thousand years somebody removed this commented code from the repository (I've been watching Lord of the rings recently:)